gallery/src/styles/NxList/NxListPage.tsx

```diff
@@ -93,17 +81,20 @@ const NxListPage = () =>
               with <code className="nx-code">.nx-list__subtext</code>
             </td>
           </tr>
+          <tr className="nx-table-row">
+            <td className="nx-cell"><code className="nx-code">.nx-list-actions</code></td>
+            <td className="nx-cell">A container for buttons inside list items</td>
+            <td className="nx-cell">Use this when you want to have a button on the far right.</td>
+          </tr>
           <tr className="nx-table-row">
             <td className="nx-cell"><code className="nx-code">.nx-list__item--empty</code></td>
             <td className="nx-cell">Modifier of <code className="nx-code">.nx-list__item</code></td>
             <td className="nx-cell">Used when there are no list items returned.</td>
           </tr>
           <tr className="nx-table-row">
-            <td className="nx-cell"><code className="nx-code">.nx-error</code></td>
+            <td className="nx-cell"><code className="nx-code">nx-list__item--error</code></td>
             <td className="nx-cell">Modifier of <code className="nx-code">.nx-list__item</code></td>
-            <td className="nx-cell">
-              Not strictly speaking a modifier, this is added to a list item when the list is in an error state.
-            </td>
+            <td className="nx-cell">This is added to a list item when the list is in an error state.</td>
```

Comment:
I think it should be mentioned that the error list item should be the only list item, and perhaps what it's expected contents are

Reply:
> I think it should be mentioned that the error list item should be the only list item, and perhaps what it's expected contents are

https://github.com/sonatype/sonatype-react-shared-components/pull/141/commits/0c923be1386ff52fae612786887ab3020728f783